src/main/java/cholog/wiseshop/db/campaign/CampaignRepository.java

```diff
@@ -12,4 +13,6 @@ public interface CampaignRepository extends JpaRepository<Campaign, Long> {
     List<Campaign> findAllByStates(List<CampaignState> states);
 
     List<Campaign> findAllByState(CampaignState campaignState);
+
+    List<Campaign> findAllByMember(Member member);

```

Comment:
이건 취향차이라서 하고싶은 방향으로해도 좋아!

개인적으로 Repository같은 인터페이스 계층에서는 JPA를 떼고 생각하는걸 좋아해서 findBy의 인자로는 최대한 객체를 안넣으려고 하는 편이야
정말 나~중에 JPA가 아니라 jdbc로 구현하게된다면 그냥 ID만 넘겨주는게 더 자연스럽지 않을까? 라는 생각이 들어서
```suggestion
    List<Campaign> findAllByMemberId(Long memberId);

```

Reply:
아~~ 그런식으로 변경지점을 생각할 수도 있구나
반영할게~!